pubspec.yaml

```diff
@@ -1,12 +1,13 @@
 name: pdfjs
 version: 2.1.266
-
 description: Dart bindings for Mozilla's PDF.js library
-
+author: Sean Burke <sean.burke@workiva.com>
 homepage: https://github.com/Workiva/pdfjs_dart
 
-authors:
-  - Sean Burke <sean.burke@workiva.com>
-
 environment:
-  sdk: '>=1.24.0 <3.0.0'
+  sdk: '>=2.4.0 <3.0.0'
+
+dev_dependencies:
```

Comment:
The `dev_dependencies` additions are probably unnecessary, since no there's currently no testing or really building of any sort happening in the library as of right now.